Dockerfile

```diff
@@ -36,21 +38,17 @@ RUN apt-get update && \
     apt-get clean -y && \
     pip install -r requirements.txt
 
-# NOTE: project files likely to change between dev builds
-COPY . /code/
 # copy built react and node libraries for production mode
 COPY --from=node-prod-deps /usr/src/app/package-lock.json /code/package-lock.json
 COPY --from=node-prod-deps /usr/src/app/webpack-stats.json /code/webpack-stats.json
 COPY --from=node-prod-deps /usr/src/app/assets /code/assets
 COPY --from=node-prod-deps /usr/src/app/node_modules /code/node_modules
 
+# NOTE: project files likely to change between dev builds
+COPY . /code/
+
```

Comment:
Why this line needed to be moved?

Reply:
The main part of this was moving the COPY of code down below the copy of the node dependencies. That takes a little time to copy that and those don't change as much as the code. You can look at this as move the 4 COPY lines for --from=node-prod-deps above this one. ;)